pkgs/development/python-modules/flask-restful/default.nix

```diff
@@ -17,6 +17,9 @@ buildPythonPackage rec {
   patchPhase = if isPy3k then ''
     rm tests/test_crypto.py tests/test_paging.py
     '' else null;
+  preCheck=''
```

Comment:
Do leave a comment here explaining what it is disabled for. Also, space at each side of `=`.

Reply:
Done.